toolkit/scss/shared_placeholders/_temporary-messages.scss

```diff
@@ -6,4 +6,13 @@
   background: $govuk-blue;
   color: $white;
   padding: $gutter-half $gutter-half 18px $gutter-half;
+
+  a, a:visited {
+    color: $white;
+  }
+
+  a:hover, a:active {
+    @extend a;
+    text-decoration: none;
```

Comment:
The links usually go from `$link-colour` to `$link-hover-colour`, which we can’t use because of the similarity in background colour. To show on hover that the underlined text is a link we could either change from `white` to `$grey-4`or `text-decoration: none;` the later was my preference, but maybe colour change would be more consistent.